esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -6370,79 +6318,129 @@ bool CWsDfuEx::onDFUFilePublish(IEspContext &context, IEspDFUFilePublishRequest
     {
```

Comment:
Need to clean up temp files on failure here.

Reply:
Yes, will add code to clearup in event of failure